modules/model/item/components/base.js

```diff
@@ -132,8 +132,8 @@ let fields = foundry.data.fields;
         data.properties = [];
         data.description.value = data.description.value || "";
         data.description.value = await TextEditor.enrichHTML(data.description.value, htmlOptions);
-        data.targetEffects = this.parent.effects.filter(e => e.application == "apply")
-        data.invokeEffects = this.parent.effects.filter(e => e.trigger == "invoke")
+        data.targetEffects = this.parent.effects.filter(e => e.application == "target")
+        data.invokeEffects = this.parent.effects.filter(e => e.trigger == "manual")
```

Comment:
These can be removed as `e.application` and `e.trigger` don't exist anymore. I'll be adding the new ways of doing this soon.